dir.common.props

```diff
@@ -72,6 +63,18 @@
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
   </PropertyGroup>
 
+  <!-- Version numbers for native binaries and build version props file
+       [ARCADE REMOVE] These properties can be removed once native binaries use Arcade versioning
+       They are included in common for packaging, which is onboarded to Arcade, but uses the version
+       numbers for native binaries for some outputs. -->
+  <PropertyGroup>
+    <NativeMajorVersion>4</NativeMajorVersion>
+    <NativeMinorVersion>6</NativeMinorVersion>
+
+    <!-- Explicitly set BuildVersionFile. This is used by BuildVersion.targets in BuildTools. -->
+    <BuildVersionFile Condition="'$(OfficialBuildId)' != ''">$(RootBinDir)obj\BuildVersion-$(OfficialBuildId).props</BuildVersionFile>
```

Comment:
The one in buildtools did, but there's no reference to this in the Arcade implementation.

Reply:
Although I just saw where we use it for our native builds, where we still have a dependency on GenerateCurrentVersion in BuildTools... This makes the FileVersion differ for native and managed bits.

Reply:
Yeah, native binaries are still using versioning from BuildTools (which is why packaging needs to get that version, not the Arcade version).

There's definitely still outstanding work around file versions in coreclr. @RussKeldorph indicated that the file versions should be `4.30.<build>` for both managed and native (which is currently not true for either in coreclr - managed is `3.0.<build>` and native is `4.6.<build>`)

Reply:
4.30? Also, there a different native version for <build> on managed and native if I'm reading this correctly - i.e. we have no correlation in versioning of the native and managed assets.

Reply:
That's what I was told, but maybe that has changed (especially since I think Arcade does minor * 100 now, so I'm not even sure how we'd get `4.30`).

Yeah, they are completely different right now...  e.g. in a build from earlier this week coreclr was `4.6.27720.71` and System.Private.CoreLib was `3.0.19.27071`

Reply:
Yeah, it should be 4.7, multiplied by arcade to 4.700.

Reply:
Sounds good. I think dealing with the versioning should be the next thing we do in the move to Arcade.